scripting/erm/ERMInterpreter.cpp

```diff
@@ -1276,10 +1276,10 @@ namespace ERMConverter
 
 			out << "ERM:addTrigger({" << std::endl;
 
-			if(!trig.identifier.is_initialized())
+			if(!trig.identifier.size())

```

Comment:
Does it have trig.identifier.empty()? Usually collections expose such method and it is a bit faster.

Reply:
Ok, using !empty() instead of size()